src/nvim/option_defs.h

```diff
@@ -743,6 +746,7 @@ enum {
   , BV_CPT
   , BV_DICT
   , BV_TSR
+  , BV_CSL
```

Comment:
This change isn't wrapped in an `#ifdef` because it prevents the generated option header from finding the `BV_CSL` symbol. It's a bit of a hack, so let me know if there's a more ideal resolution.

Reply:
I don't see any other symbol with an `#ifdef` so this is ok.